main.py

```diff
@@ -66,6 +75,58 @@ def generate_account(driver):
     time.sleep(1.5)
     driver.find_element_by_xpath(ACCEPT_BUTTON).click()
     time.sleep(2)
+    googleClass = driver.find_elements_by_class_name(CAPTCHA_BOX)[0]
```

Comment:
I'm almost wondering if a different function should be implemented for handling captchas outside of generate_account. Maybe the same code in a handle_captcha function with param driver? It's nitpicky, and the PR should still be accepted if the change isn't implemented, but I think readability could improve a bit.